src/main/java/edu/hawaii/its/groupings/access/UhAttributes.java

```diff
@@ -94,7 +65,7 @@ public List<String> getMail() {
     }
 
     public List<String> getAffiliation() {
-        return getValues("eduPersonAffiliation");
+        return getValues("eduSubjectAffiliation");
```

Comment:
The `getAffliation` should be the original `eduPersonAffiliation`. Make sure to update the test too, thanks.